src/Payments.sol

```diff
@@ -689,6 +739,13 @@ contract Payments is
         approval.rateUsage += rateIncrease;
     }
 
+    /// @notice Settles payments for a rail up to the specified epoch. Settlement may fail to reach the target epoch if either the client lacks the funds to pay up to the current epoch or the arbiter refuses to settle the entire requested range.
```

Comment:
will also fail to reach the target if it is > block number + rail lockup period

Reply:
We can't settle past the current epoch regardless.